### ISSUE TITLE
Mission request menu disappears at HQ - ded-server.

### ISSUE THREAD

**Issue by mike-187** [LordGolias/antistasi#51]
Mission request menu disappears at HQ.
To reproduce, start new game.
Go back to lobby or logoff.
Log back in.
Menu is gone at Petros.

**Comment by LordGolias** [comment]
Thanks @mike-187. Any relevant message in the log?

**Comment by mike-187** [comment]
here is the log from creating new game and re-joining the server.
https://pastebin.com/wzjGmxhD

this part happens when re-joining:
I marked bold error which I did not get before re-joining.

 8:41:21 "[AS] Server: Arsenal resynchronised"
 8:41:21 A null object passed as a target to RemoteExec(Call) 'hcsetgroup'
 8:41:26 "[AS] 2: new spawn 'fia_hq'"
 8:41:26 "[AS] Server: scheduler_fnc_execute(AS_spawn_fnc_execute):"
 8:41:26 "Worker 2: 97.8637"
 8:41:26 "Worker 3: 64.1377"
 8:41:26 "[AS] 2: spawn 'fia_hq' started state 'spawn'"
 8:41:26 Performance warning: SimpleSerialization::Write 'resources' is using type of ',TASK' which is not optimized by simple serialization, falling back to generic seriali
zation, use generic type or ask for optimizations for these types
 8:41:26 "[AS] 2: spawn 'fia_hq' finished state 'spawn'"
 8:41:26 "[AS] 2: spawn 'fia_hq' started state 'run'"
 8:41:26 Performance warning: SimpleSerialization::Read 'resources' is using type of ,'TASK' which is not optimized by simple serialization, falling back to generic serializ
ation, use generic type or ask for optimizations for these types
 **8:41:26 "[AS] Error: Client-only script AS_fnc_localCommunication called"**
 8:41:28 "[AS] 2: new spawn 'hill_[4188.99,10754.9,-156.821]'"
 8:41:28 "[AS] Server: scheduler_fnc_execute(AS_spawn_fnc_execute):"
 8:41:28 "Worker 2: 97.8139"
 8:41:28 "Worker 3: 64.1377"
 8:41:28 "[AS] 2: spawn 'hill_[4188.99,10754.9,-156.821]' started state 'spawn'"
soldier[I_Soldier_AT_F]:Some of magazines weren't stored in soldier Vest or Uniform?
soldier[I_Soldier_AT_F]:Some of magazines weren't stored in soldier Vest or Uniform?....
